controllers/machinedeployment_controller_test.go

```diff
@@ -389,6 +389,21 @@ var _ = Describe("MachineDeployment Reconciler", func() {
 			return len(machineSets.Items)
 		}, timeout*5).Should(BeEquivalentTo(0))
 
+		By("Verifying MachineDeployment has correct Conditions")
```

Comment:
Can we also try to include a negative test?

Reply:
Yes, I can do that. I will likely add it under machinedeployment_sync_test.go

Reply:
Done